src/index.ts

```diff
@@ -6,66 +6,123 @@ export * from './packages/Highlighter';
 export * from './packages/BooleanQueryTokenizer';
 export * from './packages/ThemeGenerator';
 export * from './packages/ThemeGenerator/ThemeResult';
+export * from './constants/component-specific';
+export * from './constants/keyboard';
+export * from './constants/style-related';
 
 // Atoms
-export { Button, SearchButton, StepperButton } from './components/Buttons';
-export { Callout } from './components/Callout';
-export { CandidateAvatar } from './components/CandidateAvatar';
-export { Chip } from './components/Chip';
-export { ContentPlaceholder } from './components/ContentPlaceholder';
-export { Heading } from './components/Heading';
-export { StringHighlighter, ReactElementHighlighter } from './components/Highlighter';
+export {
```

Comment:
would it make sense to use `export * from '...'` everywhere instead of listing everything one by one?

Reply:
I think yes